v23.1/create-type.md

```diff
@@ -25,32 +26,54 @@ Parameter | Description
 ----------|------------
 `type_name` | The name of the type. You can qualify the name with a [database and schema name](sql-name-resolution.html) (e.g., `db.typename`), but after the type is created, it can only be referenced from the database that contains the type.
 `IF NOT EXISTS` |  Create a new type only if a type of the same name does not already exist in the database; if one does exist, do not return an error.
-`opt_enum_val_list` | A list of values that make up the type's enumerated set.
+`opt_enum_val_list` | A list of values that make up the type's [enumerated set](#create-an-enumerated-data-type).
+`opt_composite_type_list` | A list of values that make up the set of types that make up a [composite type](#create-a-composite-data-type).
 
 ## Required privileges
 
-- To create a type, the user must have the `CREATE` [privilege](security-reference/authorization.html#managing-privileges) on the parent database.
-- To use a user-defined type in a table (e.g., when defining a column's type), the user must have the `USAGE` privilege on the type.
+- To create a type, the user must have [the `CREATE` privilege](security-reference/authorization.html#supported-privileges) on the parent database.
```

Comment:
the user also needs to have the CREATE on the schema in which the type is being created.

Reply:
updated, thanks!